### PR TITLE
Layers page

### PR DESCRIPTION
**Pull request by maithili232**
Added layers page to the website. Followed the same pattern of model page.

### CODE REVIEW

**Review by rbharath — COMMENTED** [review]
@Yukino2002 Can you review this PR?

@maithili232 Can you put up screenshots of what this looks like?

File: new-website/deepchem/styles/globals.css [hunk 1]
```diff
@@ -227,3 +228,49 @@ footer {
     @apply pb-4;
   }
 }
+
+/* layers page */
```
Comment (Yukino2002) [inline new-website/deepchem/styles/globals.css]: If you are using the exact same styling, you can just reuse the classes. No need to redeclare with a different class name.

File: new-website/deepchem/components/Layers/FilterButton.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import Image from "next/image";
```
Comment (Yukino2002) [inline new-website/deepchem/components/Layers/FilterButton.js]: This component is also exactly the same as `Models/FilterButton.jsx`. Please reuse the component, maybe move it to a directory named `Common` and use it in both models and layers page.

File: new-website/deepchem/pages/layers.js [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+/**
+ * Models component that displays the models page of the application
+ * @component
+ * @return {JSX.Element} The JSX element to render the Model component
+ */
+const Layers = () => { 
+
+  const [filteredLayers, setFilteredLayers] = useState(layers);
+  const [models, setModels] = useState([]);
```
Comment (Yukino2002) [inline new-website/deepchem/pages/layers.js:24]: I am not sure what models exactly are in the layers page, is the naming intended?

**Comment by Yukino2002** [comment]
Please add the layers page as a card in the landing page or button in the navigation bar, so that it is accessible to users.

**Comment by Yukino2002** [comment]
@maithili232 Added a few comments, will check on local once the comments have been addressed.

File: new-website/deepchem/components/Layers/LayerCard.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
+import Image from "next/image";
+import Link from "next/link";
```
Comment (Yukino2002) [inline new-website/deepchem/components/Layers/LayerCard.js:2]: This component is also very similar. I think some part has been omitted out, but I think can be reused.

**Review by Yukino2002 — CHANGES_REQUESTED** [review]

**Comment by maithili232** [comment]
> @Yukino2002 Can you review this PR?
> 
> @maithili232 Can you put up screenshots of what this looks like?

Sharing few screenshots : 
image
image
image

File: new-website/deepchem/pages/layers.js (hunk 3 above)
Comment (maithili232) [reply new-website/deepchem/pages/layers.js:24]: Models that use the layers so layers can be filtered with their respective models.

File: new-website/deepchem/components/Layers/FilterButton.js (hunk 2 above)
Comment (maithili232) [reply new-website/deepchem/components/Layers/FilterButton.js]: done.

File: new-website/deepchem/styles/globals.css (hunk 1 above)
Comment (maithili232) [reply new-website/deepchem/styles/globals.css]: done.

**Comment by maithili232** [comment]
> Please add the layers page as a card in the landing page or button in the navigation bar, so that it is accessible to users.

I have added the layers on the index page like this : 
image
I have used the datasets logo for it. Let me know if its alright or not.

**Pull request merged**